pkg/network/network.go

```diff
@@ -6,6 +6,10 @@ import (
 	"net"
 	"sort"
 	"strings"
+
+	"github.com/appgate/sdp-api-client-go/api/v17/openapi"
```

Comment:
I think this refactor is a mistake, these imports only invites further problems with circular imports, I'd suggest move it back to appliance pkg.

Reply:
which ones? be more specific about it, because the duplicate `ValidateHostnameUniqueness` function was put in a new `network` package when it was created in the first place.